README.md

```diff
@@ -44,7 +44,7 @@ const orderSigningUtils = new OrderSigningUtils()
 ### Sign, fetch, post and cancel order
 
 For clarity, let's look at the use of the API with a practical example:
-Exchanging `0.4 GNO` to `WETH` on `Goerli` network.
+Exchanging `0.4 GNO` to `WETH` on `Gnosis chain` network.
```

Comment:
Migrated examples to Gnosis chain for future proofing.
Sepolia will eventually die and we'll have to do this all over again.
It was the same with Rinkeby before Goerli.